### PR TITLE
added like and watch later functionalities

### PR DESCRIPTION
**Pull request by anasm11**
Files to check:
 liked-video-context.js
 watchlater-context.js
 LandingPage.jsx
 WatchLater.jsx
 Likes.jsx

### CODE REVIEW

File: src/contexts/liked-videos-context.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        (async()=>{const res = await axios({
+            method: 'get',
+            url: '/api/user/likes',
+            headers: {
+                authorization: localStorage.getItem('token')
+            }
+        })
+
+        console.log(res.data.likes,'init setuseeff')
+        likedVideosDispatch({ type: 'INIT', payload: res.data.likes })})()
```
Comment (yaswanthmyneni) [inline src/contexts/liked-videos-context.js:55]: changes: you can remove console.log() when you push to github

File: src/pages/LandingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import axios from "axios"
+import { useState, useEffect } from "react"
+import { IcOutlineThumbUp,IcRoundThumbUp, IcOutlineWatchLater, IcRoundWatchLater } from '../assets/index'
+import { useWatchLaterContext } from "../contexts/watchlater-context"
+import { useLikedVideosContext } from "../contexts/liked-videos-context.js"
```
Comment (yaswanthmyneni) [inline src/pages/LandingPage.jsx]: suggestion: create an index file in the contexts folder and export all contexts from it

File: src/pages/LandingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  const { watchLaterVideos, watchLaterDispatch } = useWatchLaterContext()
+  const {likedVideos,likedVideosDispatch}= useLikedVideosContext()
+
+  useEffect( () => {
+    (async()=>{
+      const res = await axios({
+        method: 'get',
+        url: '/api/videos',
+      })
+      console.log(res.data)
```
Comment (yaswanthmyneni) [inline src/pages/LandingPage.jsx]: changes: remove console logs

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks fine
Just a suggestion, UI looks incomplete, try to follow UI of figma files provided for better user experience. 👍

**Pull request merged**